_boot.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash -e
+set -o pipefail
+
+# Main directories
+readonly NODE_SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly SHIPPABLE_DIR="/etc/shippable"
+
+# Node Sub-directories
+readonly SCRIPTS_DIR="$NODE_SCRIPTS_DIR/initScripts"
```

Comment:
wouldn't it be better to move all the node-specific env related stuff in a different file and source it

Reply:
I'm not sure where I can move this. Lib?